### PR TITLE
ceramic boilerplate

### PR DESCRIPTION
**Pull request by perassijulian**
- create a conection with ceramic node
- get user DID
- check if user already has a db with contacts saved [having problems with "deterministic" method. Not fetching correctly]
- if it does compare old contacts and new contacts and add the new ones
- if it doesn't create a new db with addresses and their blank info
- save them

### CODE REVIEW

File: lib/ceramicFunctions.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
+import { CeramicClient } from '@ceramicnetwork/http-client'
```
Comment (3lLobo) [inline lib/ceramicFunctions.js:1]: Did you copy this file?
If yes, add a comment with the link on top.

File: pages/ceramic.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import Head from '../components/Head'
+import Header from '../components/Header'
+import { createAccountsDocument, updateAccountsDocument, initOrUpdateContacts } from '../lib/ceramicFunctions'
+
+const ceramic = () => {
```
Comment (3lLobo) [inline pages/ceramic.js:5]: Components are always capitalized `Ceramic`

**Review by 3lLobo — CHANGES_REQUESTED** [review]
Looks cool!
Fix the two comments and we mergeee 🚀

File: lib/ceramicFunctions.js (hunk 1 above)
Comment (perassijulian) [reply lib/ceramicFunctions.js:1]: Nope. It's just a module and it's on the package.json

File: pages/ceramic.js (hunk 2 above)
Comment (perassijulian) [reply pages/ceramic.js:5]: Even when they are pages? I can change it but it's just testing environment. When the integration works I'll delete it

File: lib/ceramicFunctions.js (hunk 1 above)
Comment (3lLobo) [reply lib/ceramicFunctions.js:1]: Then you did great work with writing explanatory comments 👌🏾

**Review by 3lLobo — APPROVED** [review]

**Pull request merged**